tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -1294,7 +1337,18 @@ private String handleFormatUnit(XPathParts parts, String unitPattern) {
                 }
             }
         }
-        return formatExampleList(examples);
+    }
+
+    /**
+     * Customize the systems according to the locale
+     *
+     * @return
+     */
+    public Set<UnitSystem> getExampleUnitSystems() {
+        LanguageTagParser ltp = new LanguageTagParser().set(cldrFile.getLocaleID());
```

Comment:
could just be `CLDRLocale.getInstance(…).getLanguage()` but that's another discussion

Reply:
Yup.